src/renderer/components/WatchAccountButton.tsx

```diff
@@ -13,13 +14,29 @@ function WatchAcountPopover(props: {
   const pubKeyVal = '';
 
   const [toKey, setToKey] = useState<string>(pubKeyVal);
+  const [validationError, setValidationErr] = useState('');
```

Comment:
I think this would be better served by a `useState<string | undefined>`. Then instead of setting it to `''` when there's no error, you can set it to `undefined`. That would make that `!!validationError` a bit more readable too.

Reply:
done